docs/ECLScheduler/ECLSched-Mods/ECLSchedComp.xml

```diff
@@ -183,6 +183,18 @@
 
         <para>To view the workunit details, click on the workunit ID (WUID)
         link for the workunit.</para>
+
+        <para>You can modify scheduled workunits from the workunit details
+        page in ECL Watch. Select the workunit details page, then press the
+        <emphasis role="bold">Reschedule</emphasis> button to change a
+        selected scheduled workunit. Press the <emphasis
```

Comment:
to change a selected scheduled workunit. 
s/b 
to reschedule a descheduled workunit.